chapters/ch07.md

```diff
@@ -1,27 +1,27 @@
 # How large do the dev/test sets need to be?
 
 ->
-# Tập phát triển / đánh giá cần lớn đến mức nào?
+# Tập phát triển/kiểm tra cần lớn đến mức nào?
 
 The dev set should be large enough to detect differences between algorithms that you are trying out. For example, if classifier A has an accuracy of 90.0% and classifier B has an accuracy of 90.1%, then a dev set of 100 examples would not be able to detect this 0.1% difference. Compared to other machine learning problems I’ve seen, a 100 example dev set is small. Dev sets with sizes from 1,000 to 10,000 examples are common. With 10,000 examples, you will have a good chance of detecting an improvement of 0.1%. [2]
 
 ->
-Tập phát triển phải đủ lớn để có thể thấy được sự khác nhau giữa các thuật toán mà bạn đang thử. Ví dụ: nếu thuật toán phân loại A có độ chính xác 90,0% và thuật toán phân loại B có độ chính xác 90,1%, thì một tập có 100 ví dụ sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập 100 ví dụ là nhỏ. Các tập phát triển với kích thước từ 1.000 đến 10.000 ví dụ khá phổ biến. Với 10.000 ví dụ, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]
+Tập phát triển phải đủ lớn để có thể thấy được sự khác nhau giữa các thuật toán mà bạn đang thử. Ví dụ: nếu thuật toán phân loại A có độ chính xác 90,0% và thuật toán phân loại B có độ chính xác 90,1%, thì một tập có 100 mẫu sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập phát triển với 100 mẫu là nhỏ. Các tập phát triển với kích thước từ 1.000 đến 10.000 mẫu khá phổ biến. Với 10.000 mẫu, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]
```

Comment:
Mình gợi ý sửa là
* Tập phát triển ~~phải~~ **_cần_** đủ lớn để ~~có thể~~ thấy được sự khác ~~nhau~~ biệt giữa các thuật toán mà bạn đang thử
Hoặc viết ngược lại là
* Để nhận ra sự khác biệt giữa các thuật toán đang thử nghiệm, thì tập phát triển phải đủ lớn. 
* Classifier --> bộ phân loại trong glossary
* thì một tập **_phát thiển_** có 100 mẫu sẽ không thể phát hiện sự khác biệt 0,1% này --> thiếu "phát triển"
* Mình nghĩ từ "vấn đề" ở đây ám chỉ các ứng dụng hoặc bài toán dùng machine learning. Nếu dịch là "vấn đề machine learning" liệu có bị hiểu nhầm sang các -- vấn đề khác như tinh chỉnh, cấu hình mạng.v.v. không? Ngoài ra có thể thêm từ "chỉ" để nhấn mạnh. 
<  So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập phát triển với **_chỉ_** 100 mẫu là nhỏ.

* Gợi ý : "Các tập phát triển thường có từ 1,000 tới 10,000 mẫu"
< Các tập phát triển với kích thước từ 1.000 đến 10.000 mẫu khá phổ biến. 

* Gợi ý: "Với một 100 mẫu thì bạn sẽ dễ cải thiện được mức 0.1% hơn."
< Với 10.000 mẫu, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]

Reply:
Cảm ơn bạn nhiều.